sites/docs/pages/components/data-table.md

```diff
@@ -97,6 +98,28 @@ This example includes a `custom_format` column, which contains a different curre
 
 <DataTable data={orders_summary} search=true/>
 
+### Sort
+
+```svelte
+<DataTable data={orders_summary} sort="sales desc">
+    <Column id=category/> 
+    <Column id=item/> 
+    <Column id=sales fmt=usd/> 
+</DataTable>
+```
+
+<DataTable data={orders_summary} sort="sales desc">
+    <Column id=category/> 
+    <Column id=item/> 
+    <Column id=sales fmt=usd/> 
+</DataTable>
+
+<DataTable data={orders_summary} sort="sales asc">
```

Comment:
@hughess Just wanted to double check, i think you wanted to remove this second sort example?